Source/WebCore/dom/DocumentFullscreen.cpp

```diff
@@ -269,10 +266,10 @@ ExceptionOr<void> DocumentFullscreen::willEnterFullscreen(Element& element, HTML
         return Exception { ExceptionCode::TypeError };
     }
 
-    // The element is an open popover.
-    if (element.isPopoverShowing()) {
-        ERROR_LOG(LOGIDENTIFIER, "Element to fullscreen is an open popover; bailing.");
-        return Exception { ExceptionCode::TypeError, "Cannot request fullscreen on an open popover."_s };
+    // FIXME: Should we enforce the iframe requirement here?
```

Comment:
Can you open a bug to track this?

Reply:
https://bugs.webkit.org/show_bug.cgi?id=288951